### DISCUSSION TITLE
Notification settings

### DISCUSSION THREAD

**Discussion by snowman-repos** [anonaddy/anonaddy#655] [Feature Requests]
I keep getting emails about failed deliveries, which I don't care about. It would be good if there was a way for me to turn these off.

**Comment by roke-julian-lockhart** [comment]
> https://github.com/anonaddy/anonaddy/discussions/655#discussion-6902799

@snowman-repos, this might be an X/Y problem. Have you read https://github.com/anonaddy/anonaddy/blob/bc626e80bfc3e21a8f9ae39fce987a33ad795e2a/README.md#can-i-mark-emails-forwarded-to-me-by-addyio-as-spam? If you're marking messages forwarded to you as spam, you might be inadvertently breaking the ToS:

> No, you must not mark messages forwarded to you by addy.io as spam as this can damage the reputation of the mail servers and is against the [terms and conditions](https://addy.io/terms/).

> addy.io is signed up to multiple feedback loops (FBLs) that trigger a notification when any messages are marked as spam. Repeatedly marking messages as spam will result in your account being disabled.

It's better to block the aliases and/or redirect messages from them to a custom folder until at least https://github.com/anonaddy/anonaddy/issues/593#issue-2127459504 is complete.

> If an alias is receiving spam messages then please deactivate it or delete it.

**Comment by snowman-repos** [comment]
ok thanks!

**Reply by roke-julian-lockhart** [reply]
> https://github.com/anonaddy/anonaddy/discussions/655#discussioncomment-10470112

@snowman-repos, glad to have been of assistance.
